tests/test_spark.py

```diff
@@ -71,13 +71,7 @@ def test_simple_schema_local_writing(self) -> None:
             spark = spark_session.getOrCreate()
             sc = spark.sparkContext
             rdd = sc.parallelize(copy.deepcopy(EXAMPLES), 100)
-            persist_wicker_dataset(
-                DATASET_NAME,
-                DATASET_VERSION,
-                SCHEMA,
-                rdd,
-                s3_storage=fake_storage,
-            )
+            persist_wicker_dataset(DATASET_NAME, DATASET_VERSION, SCHEMA, rdd, fake_storage)
```

Comment:
Are we directly testing `persist_wicker_dataset` instead of `SparkPersister.persist_wicker_dataset` to test backward compatibility?